code/API_definitions/connected-network-type.yaml

```diff
@@ -0,0 +1,416 @@
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ConnectedNetworkTypeResponse"
+              examples:
+                Connected-to-4GLTE:
+                  value:
+                    lastStatusTime: "2024-02-20T10:41:38.657Z"
+                    connectedNetworkType: 4GLTE
+                Connected-To-5GNSA:
+                  value:
+                    lastStatusTime: "2024-02-20T10:41:38.657Z"
+                    connectedNetworkType: 5GNSA
+                Connected-To-Non3gpp4g:
+                  value:
+                    lastStatusTime: "2024-02-20T10:41:38.657Z"
+                    connectedNetworkType: NON3GPP4G
+                Not-Connected:
+                  value:
+                    lastStatusTime: "2024-02-20T10:41:38.657Z"
+                    connectedNetworkType: NOT_CONNECTED
+        "400":
+          $ref: "#/components/responses/Generic400"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/ConnectedNetworkTypePermissionDenied403"
+        "404":
+          $ref: "#/components/responses/ConnectedStatusNotFound404"
+        "422":
+          $ref: "#/components/responses/ConnectedStatusUnprocessableEntity422"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "503":
+          $ref: "#/components/responses/Generic503"
+components:
+  securitySchemes:
+    openId:
+      type: openIdConnect
+      openIdConnectUrl: https://example.com/.well-known/openid-configuration
+  parameters:
+    x-correlator:
+      name: x-correlator
+      in: header
+      description: Correlation id for the different services
+      schema:
+        type: string
+  headers:
+    x-correlator:
+      description: Correlation id for the different services
+      schema:
+        type: string
+  schemas:
+    LastStatusTime:
+      description: Last time that the associated connected Network Type was updated
+      type: string
+      format: date-time
+      example: "2024-02-20T10:41:38.657Z"
+
+    ConnectedNetworkTypeResponse:
+      type: object
+      required:
+        - connectedNetworkType
+      properties:
+        lastStatusTime:
+          $ref: "#/components/schemas/LastStatusTime"
+        connectedNetworkType:
+          $ref: "#/components/schemas/ConnectedNetworkType"
+
+    ConnectedNetworkType:
+      description: |
+        * NOT_CONNECTED: The device is not connected to network
+        * 2G: 2nd Generation Mobile Communication Technology
+        * 3G: 3rd Generation Mobile Communication Technology
+        * 4GLTE: 4th Generation Mobile Communication Technology aka LTE
+        * 5GNSA:4th Generation Mobile Communication Technology aka LTE with device connected 5G New Radio (NR)
+        * 5GSA: 5th Generation Mobile Communication Technology
```

Comment:
It will be hard enough trying t work out what radio technology the device is using, without having to also work out which core it is connected to. I would suggest a single `5G` option. What is the use case where the core technology is important?